aws-connect

```diff
@@ -208,6 +224,70 @@ elif [ "${action}" == "tunnel" ]; then
     --parameters "{\"portNumber\":[\"22\"],\"localPortNumber\":[\"${port}\"]}" \
     --region "${aws_region}" \
     --profile "${aws_profile}"
+elif [ "${action}" == "document" ]; then
+
+  if [ -z "${github_token_location}" ]
+  then
+    echo "No github token provided - scripts referenced in SSM document must be in a public repo if no token is provided"
```

Comment:
I feel we can leave this echo out or it will appear for every run if you don't care about github.